File: Python/Warmup-1/front_back.py
# Given a string, return a new string where the first and last chars have been exchanged.

def front_back(str):
  if len(str) < 2:
    return str
  else:
    return str[len(str) - 1] + str[1:len(str) - 1] + str[0]

File: Java/Warmup-1/frontBack.java
/* Given a string, return a new string where the first and last chars have been exchanged. */

public String frontBack(String str) {
  if(str.length() >= 2){
    String first = str.substring(0,1);
    String last = str.substring(str.length() - 1, str.length());
    return last + str.substring(1, str.length() - 1) + first;
  } else {
    return str;
  }
}
